### DISCUSSION TITLE
HTTP/3 Support

### DISCUSSION THREAD

**Discussion by hernanrz** [OvenMediaLabs/OvenMediaEngine#1406] [Ideas]
Just wondering if there's plans to support HTTP/3 (a.k.a. QUIC)? It might be beneficial as it allows even lower latencies than http/2

**Comment by getroot** [comment]
Currently, the only HTTP-based streaming provided by OME is LLHLS. LLHLS specifies HTTP/2 in the RFC, and there is no specification for HTTP/3.

**Reply by hernanrz** [reply]
That makes total sense, thanks for your reply 👍
